test/external-encoding.js

```diff
@@ -30,5 +39,12 @@ describe('external encoding', () => {
 				expect(error.message).to.include('malformed data: URI');
 			});
 		});
+
+		it('should reject invalid data uri 2', () => {
+			return fetch('data:@@@@').catch(error => {
+				console.assert(error);
+				console.assert(error.message.includes('invalid URL'));
```

Comment:
@xxczaki while we are here, can we replace `console.assert` here with proper Chai assertions?

Reply:
@tinovyatkin I just removed this test, as with Chai assertions it would be identical to the previous one ;)

Reply:
@tinovyatkin I just removed this test, as with Chai assertions it would be identical to the previous one ;)